zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -481,7 +482,41 @@ ascan.options.strength.high.label = (More attacks)
 ascan.options.strength.insane   = Insane
 ascan.options.strength.insane.label = (MUCH more attacks)
 ascan.options.strengthTo.label		= Strength To
-ascan.options.thresholdTo.label		= Threshold To 
+ascan.options.thresholdTo.label		= Threshold To
+
+scan.filter.filtercriteria.include = Message should include one of the following {0} values: {1} 
+scan.filter.filtercriteria.exclude = Message should not include the following {0} value: {1}  
+scan.filter.filterType.Tag = Tag
+scan.filter.filterType.HttpMethod = HTTP Method
+scan.filter.filterType.HttpStatusCode = HTTP Status Code
+scan.filter.filterType.URLPattern = URL Pattern
+scan.filter.filterType.Alert = Alert
+scan.filter.filterType.ConfidenceLevel = Confidence Level
+
+scan.filter.button.apply            = Apply
+scan.filter.button.clear            = Clear
+scan.filter.button.filter           = Filter
+scan.filter.badregex.warning		   = Invalid regex: {0}
+scan.filter.desc.label.alerts = Alerts
+scan.filter.desc.label.codes = Codes
+scan.filter.desc.label.methods = Methods
+scan.filter.desc.label.tags = Tags
+scan.filter.desc.label.urlexcregex = URL Exc Regex
+scan.filter.desc.label.urlincregex = URL Inc Regex
+scan.filter.label.alerts            = Alerts:
+scan.filter.label.codes             = Codes:
+scan.filter.label.desc              = Select the required filters below. You can select multiple rows in each element. An element is not used for filtering if none of the rows in it are selected.
```

Comment:
We should add something like "only the messages that match the selections will be scanned" to be clear how the filter will work.

Reply:
ok